supabase/functions/_backend/private/validate_password_compliance.ts

```
@@ -11,6 +11,8 @@ interface ValidatePasswordCompliance {
   org_id: string
 }
 
+type OrgReadAccessResult = { allowed: boolean, error?: string }
```

ESLint

[error] Use an `interface` instead of a `type`.

(ts/consistent-type-definitions)